tests/pytests/data/LUA3107H.161/LUA3107H.161.clem_0.conf

```diff
@@ -0,0 +1,9 @@
+LEAPSECONDS_KERNEL     = /Users/ahibl/astro_efs/base/kernels/lsk/naif0012.tls
```

Comment:
Could we remove these .conf and .cmt files? They aren't necessary to get things passing

Reply:
Looks like there is still one conf, we also don't want to include the bck files